tests/ui/short_circuit_statement.fixed

```diff
@@ -6,7 +6,7 @@
 fn main() {
     if f() { g(); }
     if !f() { g(); }
-    if !(1 == 2) { g(); }
+    if 1 != 2 { g(); }
```

Comment:
Can you write tests in other cases?

Reply:
You're right, I'm going to add a test to ```sugg.rs```

Reply:
I understand. thanks!